src/CLI/klish/patches/klish-2.1.4/plugins/clish/rest_cl.cpp

```diff
@@ -106,10 +106,26 @@ int print_error(const char *str) {
         cJSON *err_msg = cJSON_GetObjectItemCaseSensitive(error, "error-message");
         if (err_msg) {
             lub_dump_printf("%% Error: %s\r\n", err_msg->valuestring);
+        /* Since error-message is an optional attribute, we need to check for "error-tag"
+           and print the error-message accordingly */
+        } else {
+            cJSON* err_tag = cJSON_GetObjectItemCaseSensitive(error, "error-tag");
+            std::string err_tag_str = std::string {err_tag->valuestring};
```

Comment:
REST server always populates error-tag in the error response. Still, I feel it is better to have a null check for err_tag.

Reply:
That's a good one. I will add that check. I missed it somehow 👍